src/components/Lobby/Lobby.js

```diff
@@ -89,12 +85,27 @@ const Lobby = () => {
   //   }
   // };
 
+  const form = useForm({
+    initialValues: {
+      // these keys must match the input keys for joinGame
+      playerName,
+      roomId
+    }
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const handleError = (_errors) => {
+    console.log('Form error handled serverside');
+  };
+  const handleSubmit = (values) => {
+    joinGame(values);
+  };
   return (
-    <>
+    <Container style={{ backgroundColor: '#d0edf5' }}>
```

Comment:
Non-blocker: you can use the `sx` prop in the future to access variables and stuff